syntaxes/gleam.tmLanguage.json

```
@@ -141,15 +219,76 @@
     "entity": {
       "patterns": [
         {
-          "begin": "\\b([[:lower:]][[:word:]]*)([[:space:]]*)?\\(",
-          "end": "\\)",
-          "patterns": [
-            {
-              "include": "$self"
+          "match": "[^\\w*](_+\\w*\\b)",
+          "captures": {
+            "1": {
+              "name": "source.ignore.gleam"
+            }
+          }
+        },
+        {
+          "match": "(\"\\w+(?:\\/*\\w+\\.{1}(?:js|mjs|cjs)+)*\") (\"\\w+\")",
+          "captures": {
+            "1": {
+              "name": "string.other.external.module.gleam"
+            },
+            "2": {
+              "name": "string.other.external.function.gleam"
+            }
+          }
+        },
+        {
+          "match": "(\\bimport\\b)(\\s\\w+)|\\/([a-z_]+)",
+          "captures": {
+            "0": {
+              "name":"meta.import.gleam"
+            },
+            "1": {
+              "name": "keyword.control.import.gleam"
+            },
+            "2": {
+              "name": "entity.name.import.gleam"
+            },
+            "3": {
+              "name": "variable.other.constant.gleam"
+            }
+          }
+        },
+        {
+          "match": "\\b((?!\\bfn\\b)[a-z][a-z_]*)\\(",
+          "captures": {
+            "1": {
+              "name": "entity.name.function.gleam"
             }
-          ],
+          }
+        },
+        {
+          "match": "(?:{| |, |\\/|\\.|\\[|\\()(\\.\\.)*(?!\\bas\\b|\\blet\\b|\\bfn\\b|\\buse\\b|\\bconst\\b|\\bcase\\b|\\btry\\b|\\bassert\\b|\\bif\\b|\\bopaque\\b|\\bexternal\\b|\\btype\\b|\\b_+\\w*\\b|\\blet _+\\w*\\b)(\\w+\\()*([a-z_]+)(?!\\/|\\(|:)\\b",
```

Since `try` is no longer a keyword:
 ```suggestion
          "match": "(?:{| |, |\\/|\\.|\\[|\\()(\\.\\.)*(?!\\bas\\b|\\blet\\b|\\bfn\\b|\\buse\\b|\\bconst\\b|\\bcase\\b|\\bassert\\b|\\bif\\b|\\bopaque\\b|\\bexternal\\b|\\btype\\b|\\b_+\\w*\\b|\\blet _+\\w*\\b)(\\w+\\()*([a-z_]+)(?!\\/|\\(|:)\\b",
```